public/src/jQuery.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function() {
+
+  bowling = new Bowling();
+
+  $('#throwBall').click(function() {
+    var score = $('#scoreChoosen').val();
+    if ( bowling.points.length <= 19) {
```

Comment:
We've got a lot of logic inside an anonymous function which is tightly coupled to our view. That makes it very difficult to test. A better approach is to name the function e.g.

```
function throwBallClicked() {
   ... code here ...
}

$(document).ready(function() {
  $('#throwBall').click(throwBallClicked);
}
```

You could then further improve it by moving the throwBallClicked function to a module or a class - either way, you are now able to test this functionality without needing to involve JQuery.